modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsaMPCv2.ts

```diff
@@ -288,12 +290,14 @@ export class EcdsaMPCv2Utils extends BaseEcdsaUtils {
     const userKeychainPromise = this.addUserKeychain(
       bitgoCommonKeychain,
       userPrivateMaterial,
+      userReducedPrivateMaterial,
       params.passphrase,
       params.originalPasscodeEncryptionCode
     );
```

Comment:
Instead of passing`userReducedPrivateMaterial`,  can we annotate the response here?

Reply:
reason of passing it here is addUserKeychain and addBackupKeychain as they exist today seem to be responsible for doing the encryption on private material so i wanted to keep all encryption operations in there for consistency